app/routes/index.js

```diff
@@ -83,6 +83,19 @@ const getModel = (data, question) => {
   }
   return model
 }
+
+function createModelNotEligible (backUrl, ineligibleContent) {
+  return {
+    backLink: backUrl,
+    messageContent: ineligibleContent.messageContent,
+    insertText: ineligibleContent.insertText,
+    messageLink: {
```

Comment:
this should be 
      messageLink: ineligibleContent.messageLink

Reply:
it is not working for title if I just write this : messageLink: ineligibleContent.messageLink